src/engine/test/assets/decoders/default_decoder.yml

```
@@ -2,8 +2,10 @@
 name: default_decoder
 
 check:
-  - queue: 1
+  - queue: +exists
+  - message: +exists
 
 normalize:
   - map:
       event.original: $message
+      wazuh.queue: $queue
```

```suggestion
  - wazuh.queue: +exists
  - event.original: +exists
```
`wazuh.queue` and `event.original` are set by the protocolHandler